### PR TITLE
Add links array to extraction

### PR DESCRIPTION
**Pull request by andjosh**
Each link object contains the `text` content and `href`.

### CODE REVIEW

**Comment by ageitgey** [comment]
Thanks for the PR. Would it be possible to add checks for link extraction to the tests?

**Comment by andjosh** [comment]
Tests added!

**Comment by andjosh** [comment]
bump

File: test/unfluff.coffee [hunk 1]
```diff
@@ -57,6 +57,18 @@ suite 'Unfluff', ->
...
         arrayEq sortedTags, dataLazy.tags().sort(), "#{site}: meta tags from partial extraction didn't match expected value"
 
+      else if field == 'links'
+        sortedLinks = data.links.sort()
+        sortedLazyLinks = dataLazy.links().sort()
+        if !orig.expected.links
+          orig.expected.links = sortedLinks
+          fs.writeFileSync("./fixtures/test_#{site}.json", JSON.stringify(orig, null, 4))
+        deepEq orig.expected.links.sort(), sortedLinks, "#{site}: links didn't match expected value"
+        # console.log(sortedLinks)
```
Comment (ageitgey) [inline test/unfluff.coffee]: Can you remove these?

Comment (ageitgey) [inline test/unfluff.coffee]: I mean the commented out console logs (the part it highlighted was a little confusing).

File: src/extractor.coffee [hunk 2]
```diff
@@ -106,6 +106,34 @@ module.exports =
 
     null
 
+  # Find any links in the doc
+  links: (doc, topNode, lang) ->
+    gravityItems = []
+    links = []
+    removeGravityItems = (gravityItems) ->
```
Comment (ageitgey) [inline src/extractor.coffee]: Why did you need to remove these nodes to find the links? Just curious what the logic is here. Does removing these items cause any side effects to the text extraction later?

Comment (andjosh) [inline src/extractor.coffee]: There were no side effects later (all tests were the same in either direction). I don't remember why I had added this loop before. After removing and running the tests again, the same results are returned without it. So let's leave it out.

**Comment by ageitgey** [comment]
Thanks!

**Comment by andjosh** [comment]
Great! When do you plan to release a new version bump?

**Comment by ageitgey** [comment]
Done! unfluff@1.1.0 should be on npm now.

**Pull request merged**